rewriter/initializer.cc

```diff
@@ -0,0 +1,128 @@
+#include "rewriter/initializer.h"
+#include "ast/Helpers.h"
+#include "ast/ast.h"
+#include "core/core.h"
+
+#include <utility>
+
+using namespace std;
+namespace sorbet::rewriter {
+
+// this only looks for sigs of the 'correct' shape (i.e. sigs with blocks that contain a send that also end with `void`)
+bool isSig(const ast::Send *send) {
```

Comment:
We have a similarly named function in `attr_reader.cc`.

What's the difference between that function and this one? Should we share the two implementations?

Reply:
They look for sigs of a slightly different shape (the one in `attr_reader` only returns true if it has a `.returns()` at the end while this one only returns true if it has a `.void()` at the end) but it would be pretty easy to unify them into a helper method.

Reply:
I've put them into a shared helper method and replace all uses of them both in `attr_reader`  and here.